pro/src/pages/Reimbursements/__specs__/ReimbursementInvoices.spec.tsx

```diff
@@ -157,4 +164,27 @@ describe('reimbursementsWithFilters', () => {
     expect(reimbursementCells[9].innerHTML).toContain('VIR4')
     expect(reimbursementCells[15].innerHTML).toContain('VIR7')
   })
+
+  it('should not display invoice banner if FF WIP_ENABLE_NEW_BANK_DETAILS_JOURNEY is off', async () => {
+    renderReimbursementsInvoices(store)
+
+    await waitForElementToBeRemoved(() => screen.queryAllByTestId('spinner'))
+    expect(
+      screen.queryByText(
+        'Les remboursements s’effectuent tous les 15 jours, rétroactivement suite à la validation d’une contremarque dans le guichet ou à la validation automatique des contremarques d’évènements. Cette page est automatiquement mise à jour à chaque remboursement.'
+      )
+    ).not.toBeInTheDocument()
+  })
+
+  it('should display invoice banner if FF WIP_ENABLE_NEW_BANK_DETAILS_JOURNEY is enable', async () => {
+    store.features.list[0].isActive = true
```

Comment:
À mon avis, ça risque de causer des problèmes en cas d'ajout de nouveau FF (en premier) dans la liste.
Ça serait plus propre de carrément faire:
```suggestion
    store.features = {
        list: [
          { isActive: true, nameKey: 'WIP_ENABLE_NEW_BANK_DETAILS_JOURNEY' },
        ],
      }
```